handbook/engineering/batch-changes/direction.md

```diff
@@ -0,0 +1,143 @@
+# Batch Changes direction
+
+Batch Changes is a tool to find code that needs to be changed and change it at scale by running code. This page outlines the vision, strategy and goals of the Batch Changes team.
+
+#### Quick links
+
+- [Planning board](https://github.com/orgs/sourcegraph/projects/119)
+- [Demo video](https://www.youtube.com/watch?v=eOmiyXIWTCw)
+- [Batch Changes documentation](https://docs.sourcegraph.com/batch_changes)
+
+# Vision
+
+#### 1 Year vision
+
+> Enterprise developers can easily create changesets across many repositories and codehosts, and track them to completion.
```

Comment:
Sorry, pet peeve (and a hill I'm prepared to die on): let's not use quoting to higlighting something. Make it bold, italic, give it a separate style, but not the quote style.